CONTRIBUTING.md

```diff
@@ -1,13 +1,13 @@
-# Contributing to Expensify.cash
-Welcome! Thanks for checking out Expensify.cash and for taking the time to contribute!
+# Contributing to Expensify
+Welcome! Thanks for checking out Expensify and for taking the time to contribute!
 
 ## Getting Started
-If you would like to become an Expensify.cash contributor, the first step is to read this document in it's entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/Expensify.cash/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, etc). Please read both documents before asking questions, as it may be covered within the documentation.
+If you would like to become an Expensify contributor, the first step is to read this document in its entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/Expensify.cash/blob/main/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, etc). Please read both documents before asking questions, as it may be covered within the documentation.
 
 #### Test Accounts
-You can create as many accounts as needed in order to test your changes directly from [expensify.cash](https://expensify.cash/). An initial account can be created when logging in for the first time, and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
+You can create as many accounts as needed in order to test your changes directly from [the app](https://expensify.cash/). An initial account can be created when logging in for the first time, and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
```

Comment:
>`[the app](https://expensify.cash/)`

The URL will be new.expensify.com. Is it premature to switch that out here until it's actually in place? We'll have a redirect on, so I guess we can just come back when it's done?

Reply:
Yeah, I had wondered about updating vs. leaving the URL until the switch had been made. I think I'll go ahead and update it since it looks like the switch will be relatively soon. We can always hold back the deployment of these changes to match up when we deploy the URL change.